go/client/logclient.go

```diff
@@ -104,15 +105,29 @@ type GetProofByHashResponse struct {
 	AuditPath [][]byte `json:"audit_path"` // An array of base64-encoded Merkle Tree nodes proving the inclusion of the chosen certificate.
 }
 
+// Logger ...
```

Comment:
Looks like this comment wasn't finished!  Since it forms part of the package document (being exported) it ought to have some useful content.